src/app.ts

```diff
@@ -10,6 +10,7 @@ import sessionMiddleware from "./session/middleware";
 import ptfSessionLoader from "./session/middleware/ptf.session";
 import {PIWIK_SITE_ID, PIWIK_URL} from "./properties";
 import authenticate from "./authentication/middleware/index";
+import companyAuthenticate from "./web.security/middleware/index";
```

Comment:
Just wondering about the package structure here. Should we maybe have './authentication/user/middleware' for the user login stuff (would mean a minor refactoring) and then place this new company authentication alongside in './authentication/company/middleware'?